packages/slate-editor/src/modules/editor/plugins/withDeserializeHtml/createDeserialize/createDeserializer.ts

```diff
@@ -22,38 +23,28 @@ export function createDeserializer(extensions: Extension[], onError: (error: unk
     const deserializeMarks = createMarksDeserializer(
         combineExtensionsMarkDeserializers(extensions),
     );
-    const deserializeText = createTextDeserializer(deserializeMarks);
+    const deserializeText = createTextDeserializer(replaceCarriageReturnWithLineFeed);
 
-    function deserialize(
-        node: HTMLElement | ChildNode,
-    ): string | Element | DeserializeHTMLChildren[] | null {
```

Comment:
Look how simpler the return type of this function has become.

Was:

```tsx
string | Element | DeserializeHTMLChildren[] | null
```

Became:

``tsx
Descendant[]
```

Great success! :D